web/src/components/core/IssuesPage.jsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useCallback, useEffect, useState } from "react";
+
+import { HelperText, HelperTextItem, Skeleton } from "@patternfly/react-core";
+
+import { partition, useCancellablePromise } from "~/utils";
+import { If, Page, Section } from "~/components/core";
+import { Icon } from "~/components/layout";
+import { useInstallerClient } from "~/context/installer";
+import { useNotification } from "~/context/notification";
+
+/**
+ * Renders an issue
+ * @component
+ *
+ * @param {object} props
+ * @param {import ("~/client/mixins").Issue} props.issue
+ */
+const IssueItem = ({ issue }) => {
+  const variant = issue.severity === "warn" ? "warning" : "error";
+  const icon = issue.severity === "warn" ? "warning" : "error";
+  const hasDetails = issue.details.length > 0;
+
+  return (
+    <div>
+      <HelperText className="issue">
+        <HelperTextItem variant={variant} hasIcon icon={<Icon name={icon} />}>
+          {issue.description}
+        </HelperTextItem>
+        <If
+          condition={hasDetails}
+          then={<HelperTextItem>{issue.details}</HelperTextItem>}
```

Comment:
NP: let's use [`pre`](https://html.spec.whatwg.org/multipage/grouping-content.html#the-pre-element) by now for honoring as mush as possible the details format (break lines and spaces)

```suggestion
          then={<HelperTextItem><pre>{issue.details}</pre></HelperTextItem>}
```